mm2src/lp_swap.rs

```diff
@@ -78,18 +78,15 @@ use std::str::FromStr;
 use std::sync::{Arc, Mutex, Weak};
 use uuid::Uuid;
 
-#[path = "lp_swap/maker_swap.rs"] mod maker_swap;
-
-#[path = "lp_swap/taker_swap.rs"] mod taker_swap;
-
-#[path = "lp_swap/pubkey_banning.rs"] mod pubkey_banning;
-
 #[path = "lp_swap/check_balance.rs"] mod check_balance;
-#[path = "lp_swap/trade_preimage.rs"] mod trade_preimage;
-
+#[path = "lp_swap/maker_swap.rs"] mod maker_swap;
 #[path = "lp_swap/my_swaps_storage.rs"] mod my_swaps_storage;
+#[path = "lp_swap/pubkey_banning.rs"] mod pubkey_banning;
+#[path = "lp_swap/recover_swap.rs"] mod recover_swap;
```

Comment:
It's worth changing module and file names too.

Reply:
Done